docs/guides/functions.ipynb

```diff
@@ -49,52 +49,94 @@
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "id": "be135691-adee-4a1c-98bf-e76deb12319f",
+   "execution_count": null,
+   "id": "24dc565f-bb5d-4164-ba44-fc04c2fcaafd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "token = \"<your-token>\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32919d16-4b5e-439a-8cda-33687a3d96b4",
    "metadata": {},
    "outputs": [],
    "source": [
     "from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
     "\n",
-    "catalog = QiskitFunctionsCatalog(token=\"<YOUR_IQP_API_TOKEN>\")"
+    "QiskitFunctionsCatalog.save_account(token=token)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "42843fcd-1779-4a74-8d6a-7d00c22e374e",
+   "id": "e458aad5-fa7c-4ba1-8e9e-be770d89a04a",
    "metadata": {},
    "source": [
-    "Optionally, you can use the `save_account()` method to save your credentials for easy access later on, before initializing the service. This saves your credentials in the same place as `QiskitRuntimeService.save_account()`, so you can skip this step if you had previously used `QiskitRuntimeService` to save your account. *Note that account credentials are saved in plain text, so only do so if you are using a trusted device.*"
+    "Close out of the REPL with `exit()`. From now on, whenever you need to authenticate to the service, you can load your credentials with `QiskitFunctionsCatalog()`."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "54fc48c8-3a65-478c-9bc3-ec6eb4db425a",
+   "id": "e872c222-aee9-40eb-8778-9a8e5434a11c",
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
+    "# Load saved credentials\n",
+    "catalog = QiskitFunctionsCatalog()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80441467-1d9a-4107-8570-22afa6e369f7",
+   "metadata": {},
+   "source": [
+    "3. <span id=\"functions-untrusted\"></span>**Avoid executing code on an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by rotating it on the [IBM Quantum Platform dashboard](https://quantum.ibm.com/) (click the refresh button in the API token field). To initialize the service in this situation, expand the following section to view code you can use:\n",
+    "\n",
+    "    <details>\n",
+    "        <summary>Initialize the service in an untrusted environment</summary>\n",
+    "\n",
+    "    ```python\n",
+    "    from qiskit-ibm-catalog import QiskitFunctionsCatalog\n",
     "\n",
-    "# This can be skipped if you previously did QiskitRuntimeService.save_account()\n",
-    "QiskitFunctionsCatalog.save_account(token=\"<YOUR_IQP_API_TOKEN>\")"
+    "    # Refresh your API token on the dashboard after using the code as follows:\n",
```

Comment:
I think people may find this comment confusing because they may think the line of code is doing the refreshing. Instead, they need to go into the dashboard to refresh their token. Maybe use normal text outside the code to explain that? 

> Warning: you should expire your token after closing your session on an untrusted machine to reduce risk. Go to the [instructions on how to refresh...].

Reply:
There was already some text outside the code but I strengthened it a bit and swapped in the word "expire" for "rotate" since I think that's more accurate. Also rewrote the actual code comment so there's no confusion about the code doing the expiring on its own!